package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -793,27 +1090,17 @@ def _reformat_hb(hb, atomformat="{0[0]!s}{0[1]!s}:{0[2]!s}"):
     def timeseries(self):
         r'''Time series of water bridges.
 
-        Due to the intrinsic complexity of water bridge problem, where several
```

Comment:
Is the output data structure still explained elsewhere?

Reply:
@orbeckst I found that this complex explanation confuses people and, as a result, the timeseries attribute has been modified to a new simplified structure, which is explained in output section.